dough-protection/src/main/java/io/github/bakedlibs/dough/protection/modules/RedProtectProtectionModule.java

```diff
@@ -46,7 +49,15 @@ public boolean hasPermission(OfflinePlayer p, Location l, Interaction action) {
         Player player = (Player) p;
         switch (action) {
             case INTERACT_BLOCK:
-                return region.canChest(player);
+                if (l.getBlock() instanceof Container){
```

Comment:
`Container` is an extension of `BlockState`, only the block's state can therefore be an instance of `Container`.